lib/helpers/index.ts

```diff
@@ -1,8 +1,12 @@
-import * as Bluebird from 'bluebird';
-import * as path from 'path';
-import * as express from 'express';
 import * as sdk from 'etcher-sdk';
 
+export function promiseStream(stream: NodeJS.WritableStream): Promise<void> {
```

Comment:
```suggestion
export function promiseStream(stream: Stream.Stream): Promise<void> {
```
With `import * as Stream from 'stream';` above.